core/src/main/java/nu/yona/server/analysis/entities/IntervalActivity.java

```diff
@@ -55,8 +55,8 @@ public static IntervalActivityRepository getIntervalActivityRepository()
 	 */
 	private LocalDate startDate;
 
-	@ElementCollection
-	private List<Integer> spread;
+	@Column(length = SPREAD_COUNT)
```

Comment:
Currently, `spread` is never actually set because the aggregates are never computed (there is no batch script yet).